agent/command/git_test.go

```diff
@@ -325,9 +325,9 @@ func (s *GitGetProjectSuite) TestTokenScrubbedFromLogger() {
 	s.Require().NoError(err)
 	conf.ProjectRef.Repo = "invalidRepo"
 	conf.Distro = nil
-	s.comm.CreateInstallationTokenFail = true
 	token := "abcdefghij"
 	conf.Expansions.Put(evergreen.GlobalGitHubTokenExpansion, token)
+	s.comm.CreateInstallationTokenResult = token
```

Comment:
This test tests to make sure the token is scrubbed, this just passes through the token to make sure